openmetadata-ui/src/main/resources/ui/package.json

```diff
@@ -104,11 +104,11 @@
     "preinstall": "cd ../../../../.. && yarn install --frozen-lockfile",
     "postinstall": "yarn run build-check",
     "pre-commit": "lint-staged --concurrent false",
-    "test": "jest --passWithNoTests --maxWorkers=3 ",
+    "test": "jest --passWithNoTests --maxWorkers=3 --silent",
```

Comment:
Let's just add silent for this script only, since it's using in Maven-CI and build process: https://github.com/open-metadata/OpenMetadata/blob/ed884cf79aee35241fdf6d07fd9ce62e9175102b/openmetadata-ui/pom.xml#L131

we can reduce errors with watch and other scripts to fix them

Reply:
@chirag-madlani , Done.